modules/core/04-channel/v2/client/cli/cli.go

```diff
@@ -36,7 +36,8 @@ func NewTxCmd() *cobra.Command {
 	}
 
 	txCmd.AddCommand(
-		newProvideCounterpartyCmd(),
+		newCreateChannelTxCmd(),
+		newProvideCounterpartyTxCmd(),
```

Comment:
in the spec we're calling this register counterparty? should we open a small nit PR to do renaming? 

happy to pick it up